src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
```

Comment:
```suggestion
  const calculator = {
```
називаємо змінні більш змістовно)